### ISSUE TITLE
SSL settings are not applied

### ISSUE THREAD

**Issue by pikausp** [NginxProxyManager/nginx-proxy-manager#184]
**Describe the bug**
SSL settings are not applied when creating a new proxy. The cert is indeed generated and HTTPS is supported, however, settings such as Force SSL do not take effect. I found out when I added a new proxy and tried accessing the site. I was not redirected to HTTPS. 

I verified I had not forgotten to check toggle the buttons by creating a new site and verified the same issue occurred. I also checked the `id.conf` file and things such as including the force SSL file were missing.

Opening the `Edit` dialog and toggling the options gets the desired results.

The version of NPM I use is 2.0.13

This is the conf file when I checked Force SSL, HTTP/2 Support and HSTS Enabled.

```
server {
  set $forward_scheme http;
  set $server         "127.0.0.1";
  set $port           81;

  listen 80;
listen 443 ssl http2;

  server_name my.test.domain.com;


  # Let's Encrypt SSL
  include conf.d/include/letsencrypt-acme-challenge.conf;
  include conf.d/include/ssl-ciphers.conf;
  ssl_certificate /etc/letsencrypt/live/npm-4/fullchain.pem;
  ssl_certificate_key /etc/letsencrypt/live/npm-4/privkey.pem;

# Asset Caching
  include conf.d/include/assets.conf;

  # Block Exploits
  include conf.d/include/block-exploits.conf;

  access_log /data/logs/proxy_host-4.log proxy;

  location / {
    # Proxy!
    include conf.d/include/proxy.conf;
  }
}
```

**To Reproduce**
1. Go to 'Proxy Hosts'
2. Click on 'Add proxy host'
3. Fill out the site information
4. Go to 'SSL' tab
5. Toggle on Force SSL, HTTP/2 Support, HSTS Enabled

**Expected behavior**
The settings will take effect and the site config will reflect the settings.

**Screenshots**
When I select all those and then go edit the host, the certificate is selected but all the options I checked previously are unchecked 
![image](https://user-images.githubusercontent.com/6717146/62621559-72566c00-b91c-11e9-8787-bca799d03c4b.png)

**Comment by nldoty** [comment]
I am also experiencing this as well.

**Comment by Giom-fm** [comment]
Got the same problem over here.

**Comment by Giom-fm** [comment]
Ok wait. After restarting the Browser everything work fine now.

**Comment by chris1668** [comment]
Yes that was my other problem, Firefox's web cache did not play nice with
the upgrade and once I cleared it the rest of Web Interface would load.

**Comment by tophee** [comment]
I am also having this issue on NPM 2.3.1

My settings:
![image](https://user-images.githubusercontent.com/3662750/103365015-65ad5780-4abf-11eb-8d05-a0720cd3b6b3.png)

But I can still consistently and on different browsers connect via http. It does not enforce https.

**Comment by beefycheddar** [comment]
I am experiencing this on 2.7.1. Https is not enforced even with the switch enabled.

**Comment by dnv** [comment]
Having the same issue on v2.7.3, Force SSL doesn't actually enforce a redirect. Reset your entire browser cache/cookies/etc and try accessing via http directly, it will work even though it shouldn't. If you browser is aware of the https also working it can redirect you without asking (at least Chrome seemingly does), but it's the browser doing the redirect, not NPM. Can be confirmed by resetting the browser.

**Comment by ma-karai** [comment]
is there an update or workaround on this ?

**Comment by dnv** [comment]
> is there an update or workaround on this ?

My workaround was migrating to Caddy.

**Comment by adrianhelvik** [comment]
I'm also faced with this issue. SSL is simply not working.

**Comment by ma-karai** [comment]
ok

**Comment by chaptergy** [comment]
Possibly related: https://github.com/jc21/nginx-proxy-manager/issues/516

**Comment by ma-karai** [comment]
is this fixed ?

**Comment by oaeide** [comment]
Seems this is still an issue. Opened 07.aug 2019...what?

**Comment by zierbeek** [comment]
still an issue;...

**Comment by zierbeek** [comment]
is the team still on active development even?

**Comment by jackblk** [comment]
Still not fixed, but can be circumvent:

* Create a new proxy host/redirection
* Edit the proxy: enable Force SSL
* Clear browser cache/data (or just open an incognito/private session)

It should be redirected to HTTPS now.

I think this is the same issue with https://github.com/NginxProxyManager/nginx-proxy-manager/issues/516

**Comment by realies** [comment]
This is still happening.

**Comment by realies** [comment]
> Issue is now considered stale. If you want to keep it open, please comment :+1:

brah.

**Comment by studio-outsider** [comment]
Still an issue

**Comment by Visualflea** [comment]
I'm seeing this issue

**Comment by delta0815** [comment]
Same issue here

**Comment by Am0rphous** [comment]
Same issue.

docker: image: 'jc21/nginx-proxy-manager:latest
DB:  image: 'postgres:17'
Ubuntu server.
